.snippets/code/node-operators/networks/run-a-node/systemd/terminal/logs.md

```diff
@@ -0,0 +1,10 @@
+<div id="termynal" data-termynal>
+    <span data-ty>Feb 03 03:35:33 ubuntu-jammy moonbase[52847]: 2025-02-03 03:35:33 [Relaychain] @ Syncing 158.7 bps, target=#12361301 (9 peers), best: #35487 (0x527e...c72b), finalized #35328 (0xfcd2...828), # 239.9kiB/s 15.6kiB/s</span>
```

Comment:
There shouldn't be the ubuntu-jammy in the actual span, it should be added through the use of `<span data-ty="input">`